### DISCUSSION TITLE
Document Import not working

### DISCUSSION THREAD

**Discussion by bmo-at** [jonaswinkler/paperless-ng#782] [Support] [answered]
Hi, hope you guys can help me. I am trying to migrate my paperless-ng documents saved in one instance to another one on a different machine.

The import of the documents fails however with the following exceptio from Python, indicating that the 'colour' property on a Tag of some kind is missing:

File "/usr/local/lib/python3.7/site-packages/django/db/models/options.py", line 577, in get_field
    raise FieldDoesNotExist("%s has no field named '%s'" % (self.object_name, field_name))
django.core.exceptions.FieldDoesNotExist: Tag has no field named 'colour'

But looking at the manifest.json, all the document.Tags seem to have a colour property associtated with them.

**Comment by jonaswinkler** [comment, marked as answer]
Make sure both instances of paperless are on the same version. The import only really works on versions it was exported on.

"colour" got removed in 1.3.0 in favor of "color", which allows custom tag colors instead of a predefined palette of 16 colors. The importer indicates that this field does not exist anymore.

**Reply by bmo-at** [reply]
Alright, sounds like a solid solution. The exporting instance is indeed on 1.2.1
I'm gonna assume that's it

**Reply by jonaswinkler** [reply]
Use 1.2.1 on your target instance then, import the data, then update.

**Reply by bmo-at** [reply]
Let me take this opportunity to thank you for this great continuation of a cool project!
